instance/tests/views/test_index.py

```diff
@@ -36,7 +36,7 @@ class IndexViewsTestCase(WithUserTestCase):
     """
     url = reverse('instance:index')
     login_url = reverse(settings.LOGIN_URL)
-    register_url = reverse('registration:register')
+    register_url = login_url
```

Comment:
@aayushagra If the user doesn't have staff permissions, he should be redirected to the new console.
If he's logged out, to the login form. See `test_index_authenticated_basic_user` below.